test/e2e/scale/azdisk_scale_test.go

```diff
@@ -51,29 +52,31 @@ func scaleTests(isMultiZone bool) {
 
 	ginkgo.It("Scale test scheduling and starting multiple pods with a persistent volume.", func() {
 		e2eutils.SkipIfUsingInTreeVolumePlugin()
-		e2eutils.SkipIfNotUsingCSIDriverV2()
-
-		volume := testsuites.VolumeDetails{
-			FSType:    "ext4",
-			ClaimSize: "256Gi",
-			VolumeMount: testsuites.VolumeMountDetails{
-				NameGenerate:      "test-volume-",
-				MountPathGenerate: "/mnt/test-",
-			},
+		volumes := []testsuites.VolumeDetails{}
+		for j := 1; j <= 1; j++ {
+			volume := testsuites.VolumeDetails{
```

Comment:
I am assuming this is an old change which overlapped my recent fixes. I dont see a need to run a loop from 1 to 1.

Reply:
The only reason I keep the loop is because it allows us to change the number of volumes per pod by just updating the loop counter. We can remove it if we know we only want to run the test with one volume.

Reply:
There is another test case with multiple volumes being attached to a pod. So, I would rather clear up this change to reduce confusion.